src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. It should point to a compiled CSS file instead. Ensure that the SCSS is compiled to CSS and link to the CSS file.